openupgrade_scripts/scripts/account/15.0.1.2/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,199 @@
+---Models in module 'account'---
+new model account.accrued.orders.wizard [transient]
+new model account.payment.method.line
+new model account.tax.carryover.line
+# NOTHING TO DO: new model
+
+---Fields in module 'account'---
+account      / account.account          / message_follower_ids (one2many): NEW relation: mail.followers
+account      / account.account          / message_ids (one2many)        : NEW relation: mail.message
+account      / account.account          / message_main_attachment_id (many2one): NEW relation: ir.attachment
+account      / account.account          / website_message_ids (one2many): NEW relation: mail.message
+# NOTHING TO DO: new feature
+
+account      / account.account.tag      / applicability (selection)     : selection_keys is now '['accounts', 'products', 'taxes']' ('['accounts', 'taxes']')
+# NOTHING TO DO: new feature
+
+account      / account.account.template / message_follower_ids (one2many): NEW relation: mail.followers
+account      / account.account.template / message_ids (one2many)        : NEW relation: mail.message
+account      / account.account.template / message_main_attachment_id (many2one): NEW relation: ir.attachment
+account      / account.account.template / website_message_ids (one2many): NEW relation: mail.message
+# NOTHING TO DO: new feature
+
+account      / account.analytic.line    / category (False)              : NEW selection_keys: ['invoice', 'other', 'vendor_bill'], mode: modify
+# NOTHING TO DO: new feature
```

Comment:
@tranngocson1996 chỗ này cần phải điền giá trị cho nó
Dựa vào `move_id của account.analytic.line` để điền giá trị, nếu `move_type in ['out_invoice', 'out_refund', 'out_receipt']` thì là `invoice`
Nếu  `move_type in ['in_invoice', 'in_refund', 'in_receipt']` thì là `vendor_bill`
Còn lại là `other`